markdownhighlighter.cpp

```diff
@@ -115,6 +115,9 @@ void MarkdownHighlighter::initHighlightingRules() {
     {
         HighlightingRule rule(HighlighterState::MaskedSyntax);
         rule.pattern = QRegularExpression(QStringLiteral(R"(^\[.+?\]: \w+://.+$)"));
+        rule.shouldContain[0] = QStringLiteral("://");
+        rule.shouldContain[1] = QLatin1String("");
```

Comment:
do you really need to set empty strings if you don't need them? in my tests these lines also could be let out

Reply:
I am trying another way. It is faster. And it doesn't contain these.

They aren't necessary though.